README.md

```diff
@@ -15,6 +15,27 @@ Parameters as array by name (not by in)
 
 check application of proper nouns
 
+### Security
+
+Not sure if that declaration at line 20 works globally, i.e. may need to apply at each endpoint (see userLogin).
```

Comment:
The Security setting becomes the default for all endpoints ... 
```
security:
  - authToken: [] 
```
UNLESS overridden for a particular endpoint.  `userLogin` should have no security, so 
`security: []`

Reply:
that makes sense -- just the swagger linter did not like the existing format, will see what I can figure out and return login to none